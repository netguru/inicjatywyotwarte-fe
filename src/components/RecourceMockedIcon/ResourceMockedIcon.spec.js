import React from 'react'
import ResourceMockedIcon from './RecourceMockedIcon'
import { shallow, mount } from 'enzyme';
import toJson from 'enzyme-to-json';

describe('ResourceMockedIcon', () => {
  const [title, category, size] = ['Title', 'neighbourly_help', 50];

  test('renders mocked icon with proper title', () => {
    const wrapper = shallow(
      <ResourceMockedIcon
        title={title}
        category={category}
        size={size}
      />
      );
    expect(wrapper.find('Icon').length).toEqual(1);
    expect(wrapper.text()).toEqual('Title');
  });

  test('match snapshot', () => {
    let wrapper = shallow(
      <ResourceMockedIcon
        title={title}
        category={category}
        size={size}
      />
    );

    expect(toJson(wrapper)).toMatchSnapshot();
  });
});


