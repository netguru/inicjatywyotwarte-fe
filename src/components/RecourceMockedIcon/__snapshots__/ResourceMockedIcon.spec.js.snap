// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResourceMockedIcon match snapshot 1`] = `
<Icon
  size={50}
>
  Title
</Icon>
`;
